HTML - 38/memorizeTheOrderOfRedCards.js

```diff
@@ -0,0 +1,75 @@
+let userCombination = [];
+let combination = randomArray();
+
+function randomArray() {
+    let a=[];
+    for (let i=0;i<30;++i) {
+        a[i]=i;
+    }
+    function shuffle(array) {
+        let tmp, current, top = array.length;
+        if(top) while(--top) {
+            current = Math.floor(Math.random() * (top + 1));
+            tmp = array[current];
+            array[current] = array[top];
+            array[top] = tmp;
+        }
+        return array;
+    }
+    a = shuffle(a);
+    let arr = [a[1], a[2], a[3], a[4]];
+    return arr;
+}
+
+async function start() {
+    if (userCombination.length > 0) {
+        for (let i = 0; i < combination.length; i++) {
+            document.getElementById(combination[i].toString()).innerHTML = "";
+            document.getElementById(combination[i].toString()).style.backgroundColor = "#267326";

```

Comment:
Like magic number there are magic strings too, this colors are example. 
You need to put them in variable. Variable name will explain what is inside.
When you have multiple variables you can create config variable like:
`const colors={correct:'green',wrong:'red',closed:'gray'}` and use colors.correct when calling it.

Reply:
I noticed you used word "magic" a few times. What does that mean?
In this example I should put "document.getElementById(combination[i].toString()).colors.green;"?